xml/System/Random.xml

```diff
@@ -582,13 +584,13 @@ The following example uses the parameterless constructor to instantiate three <x
  The following example generates random integers with various overloads of the <xref:System.Random.Next%2A> method.  
```

Comment:
this page doesn't have the note. Can you please add it to the MemberGroup element similar to what I did here?
https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/TimeSpan.xml#L2809

Reply:
Done.